javascript/react-js/inputs-and-lists.md

```diff
@@ -84,19 +86,13 @@ class App extends Component {
     const { task, tasks } = this.state;
 
     return (
-      <div className="col-6 mx-auto mt-5">
-        <form>
-          <div className="form-group">
-            <label htmlFor="taskInput">Enter task</label>
-            <input type="text" id="taskInput" className="form-control" />
-          </div>
-          <div className="form-group">
-            <button type="submit" className="btn btn-primary">
-              Add Task
-            </button>
-          </div>
-        </form>
-      </div>
+      <form>
```

Comment:
One last small thing, I realized that we need a root div because we add the Overview component to the App component later on- you can just add that root div back up here so it's more clear. Sorry about that! But thank you for removing the other unnecessary divs! I'll merge this once this is updated. You can just push another commit, instead of opening a new PR.

Reply:
I have just added the root div back. 

Don't be sorry, I'm happy to help. :)